app/tests/integration/test_05_tokeninfo.py

```
@@ -0,0 +1,401 @@
+    """
+    client = TokenInfoEndpointClient()
+
+    # Use a test API key in GW2 format (similar to real API keys)
+    test_api_key = "AAAAAAAA-BBBB-CCCC-DDDD-EEEEEEEEEEEEFFFFFFFF-1111-2222-3333-444444444444"
+
+    response = await client.get_tokeninfo(test_api_key)
+
+    assert response.status_code == 200, \
+        f"Expected status code 200, got {response.status_code}: {response.text}"
+
+    print(f"✅ /tokeninfo endpoint responded successfully with valid API key")
```

Ruff (0.14.3)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)